### PR TITLE
*Update load application list from myapps url in the Designer page

### PR DESCRIPTION
**Pull request by wongpratan**
https://github.com/digi-serve/ns_app/issues/314

## Release Notes

- Update load application list from myapps url in the Designer page to improve performance

### CODE REVIEW

**Review by guyyoo — APPROVED** [review]
LGTM.

File: src/rootPages/Designer/ui_choose_list.js [hunk 1]
```diff
@@ -351,9 +351,20 @@ export default function (AB) {
...
+
+            if (this.AB.applications((app) => app.id == def.id).length < 1) {
+               // Add a new Application to AB._allApplications
+               this.AB.definitionSync("created", def.id, def);
+            }
+
+            this.loadData();
+         };
+
          this._handler_reload = (def) => {
```
Comment (nh758) [inline src/rootPages/Designer/ui_choose_list.js:365]: do we still need this function?

**Review by nh758 — APPROVED** [review]
Make sure to add a label, so that the build works when you merge.

File: src/rootPages/Designer/ui_choose_list.js (hunk 1 above)
Comment (wongpratan) [reply src/rootPages/Designer/ui_choose_list.js:365]: Yes, I should rename that function to clarify

**Pull request merged**